S3/English/ASS_29.ass

```diff
@@ -0,0 +1,105 @@
+﻿[Script Info]
+; Script generated by Aegisub 3.2.2
+; http://www.aegisub.org/
+ScriptType: v4.00+
+WrapStyle: 0
+ScaledBorderAndShadow: yes
+YCbCr Matrix: TV.601
+PlayResX: 1920
+PlayResY: 1080
+
+[Aegisub Project Garbage]
+Audio File: 万圣街_1920x1080_万圣街_29.mp4
+Video File: 万圣街_1920x1080_万圣街_29.mp4
+Video AR Mode: 4
+Video AR Value: 1.777778
+Video Zoom Percent: 0.375000
+Active Line: 10
+Video Position: 2093
+
+[V4+ Styles]
+Format: Name, Fontname, Fontsize, PrimaryColour, SecondaryColour, OutlineColour, BackColour, Bold, Italic, Underline, StrikeOut, ScaleX, ScaleY, Spacing, Angle, BorderStyle, Outline, Shadow, Alignment, MarginL, MarginR, MarginV, Encoding
+Style: Default,Arial,45,&H00FFFFFF,&H000000FF,&H00000000,&H00000000,0,0,0,0,100,100,0,0,1,2,2,2,18,0,20,1
+Style: Title,Arial,75,&H006CC2E9,&H000000FF,&H00000000,&H00000000,0,0,0,0,100,100,0,0,1,2,2,2,10,10,280,1
+
+[Events]
+Format: Layer, Start, End, Style, Name, MarginL, MarginR, MarginV, Effect, Text
+Dialogue: 0,0:01:02.51,0:01:05.56,Title,,0,0,0,, {\pos(970,920)} {\b1\an5\pos(970,880)\fs120\frz4.5\t(0,100,1.5\fs45)}Episode 29:  {\c&H7990E7&}Be aware of stray cats{\b0}
+Dialogue: 0,0:01:10.15,0:01:11.75,Default,,0,0,0,,Stop barking, move!
+Dialogue: 0,0:01:11.75,0:01:12.18,Default,,0,0,0,,Move!
+Dialogue: 0,0:01:17.00,0:01:17.80,Default,,0,0,0,,Is this...?
+Dialogue: 0,0:01:19.56,0:01:20.27,Default,,0,0,0,,Ira
+Dialogue: 0,0:01:20.38,0:01:21.02,Default,,0,0,0,,Look at this
+Dialogue: 0,0:01:21.65,0:01:23.20,Default,,0,0,0,,I found a cat downstairs
+Dialogue: 0,0:01:23.48,0:01:24.60,Default,,0,0,0,,It has two tails
+Dialogue: 0,0:01:24.85,0:01:25.62,Default,,0,0,0,,So unusual
+Dialogue: 0,0:01:25.93,0:01:26.84,Default,,0,0,0,,Isn't it...?
+Dialogue: 0,0:01:27.20,0:01:28.17,Default,,0,0,0,,I want to raise it
+Dialogue: 0,0:01:28.60,0:01:30.73,Default,,0,0,0,,But not sure if Lynn allows it
+Dialogue: 0,0:01:30.10,0:01:31.32,Default,,0,0,0,,{\pos(970,200)}What a great smell of alcohol
```

Comment:
"great" can be kept I think, I personally would go more to "What a strong smell of alcohol"?